docs/catalog.md

```diff
@@ -529,3 +528,7 @@ The following is an example of creating a trigger that will be fired each time t
   ```
   $ wsk trigger create myGitTrigger --feed myGit/webhook --param events push
   ```
+
+When you `push` a commit into the Github repository the trigger will be fired by the webhook, any action associated with a rule with the trigger will be invoked
```

Comment:
A commit to the Github repository via a `git push` will cause the trigger to be fired by the webhook. If there is a rule that matches the trigger, then the associated action will be invoke. The action receives the Github webhook payload as an input parameter. Each Github webhook event has a similar JSON schema, but a unique `payload` object that is determined by its event type.